src/main/scala/org/scalajs/tools/tsimporter/Main.scala

```diff
@@ -31,14 +31,24 @@ object Main {
     val outputFileName = args(1)
     val outputPackage = if (args.length > 2) args(2) else "importedjs"
 
-    val definitions = parseDefinitions(readerForFile(inputFileName))
+    if (!importTsFile(inputFileName, outputFileName, outputPackage)) {
+      System.exit(2)
+    }
+}
 
-    val output = new PrintWriter(new BufferedWriter(
-        new FileWriter(outputFileName)))
-    try {
-      process(definitions, output, outputPackage)
-    } finally {
-      output.close()
+  def importTsFile(inputFileName: String, outputFileName: String, outputPackage: String): Boolean = {
```

Comment:
Similarly, `importTsFile` should return an `Either[String, Unit]`. That way `ImporterSpec` would be able to do something like `assertEquals(Right(()), result)`, which would nicely display the offending error message, associated with the failing test, in case of a parse error.

Currently, in case there is a parse error in a test, the `Console.err.println()` will print it in a way that is dissociated from any specific test.